src/server/routes/revision.js

```diff
@@ -58,8 +58,8 @@ function formatAuthorChange(change) {
 	if (_.isEqual(change.path, ['ended'])) {
 		return baseFormatter.formatEndedChange(change);
 	}
-
-	if (_.isEqual(change.path, ['type'])) {
+	if (_.isEqual(change.path, ['authorType']) ||
+			_.isEqual(change.path, ['authorType', 'label'])) {
```

Comment:
I wonder if this would be simpler and essentially work the same; what do you think?
This would apply to the other entity types as well.
```suggestion
	if (change.path?.[0] === 'authorType']) {
```

Reply:
this will also match the path `['authorType',  'id' ]` leading to a Type Error.

![Screenshot from 2021-07-15 19-44-59](https://user-images.githubusercontent.com/55311336/125803583-8f102c93-8189-4aa1-b374-22cfd114c024.png)

Reply:
Ah yes, thanks :)